algebra/group_power.lean

```diff
@@ -304,6 +304,20 @@ theorem add_monoid.smul_nonneg [ordered_comm_monoid α] {a : α} (H : 0 ≤ a) :
 | 0     := le_refl _
 | (n+1) := add_nonneg' H (add_monoid.smul_nonneg n)
 
+lemma pow_abs [decidable_linear_ordered_comm_ring α] (a : α) (n) : (abs a)^n = abs (a^n) := 
+begin 
```

Comment:
Can you write the proof in one line, as I wrote in a previous comment? Or is there something failing / missing?

Reply:
Not sure how you mean, I can prove it in the same way, but put it all in one line, but it does need induction.

Reply:
Do you mean like this `lemma pow_inv [discrete_field α] (a : α) (n) : (a^n)⁻¹ = (a⁻¹)^n :=
begin 
  induction n with n hi, 
  all_goals { simp [*, monoid.pow, mul_inv', mul_comm] }
end

Reply:
Yes, just written differently:
`by induction n; simp [*, monoid.pow, mul_inv', mul_comm]`
This exactly the same, but as these kind of proofs are very common they should be written in the same style.
When it is written in the form `begin induction _, all_goals { ... } end ` I need to look at the different pieces and think if it is really the same or if there is a slight difference.